### ISSUE TITLE
blue and yellow boxes in 11Q1 

### ISSUE THREAD

**Issue by oritRosengarten** [Scripta-Qumranica-Electronica/ScrollEditor#257]
Any idea on what is causing the problem (I attached a screen shot) and how to fix it
![Capture bug11Q1](https://user-images.githubusercontent.com/74494426/135404151-14cefab0-ecd9-4525-98dd-c6d86fb32ea8.JPG)
?

**Comment by Bronson-Brown-deVost** [comment]
Yes, the automatic artefact isolation algorithm grabbed little pieces of the color card instead of the manuscript fragment. I can try to jog it into making some better choices there.

Bronson
